sdk/core/System.BinaryData/src/System.BinaryData.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Contains the BinaryData which is useful for converting between string, stream, Json, and bytes.</Description>
+    <AssemblyTitle>System.BinaryData</AssemblyTitle>
+    <Version>1.0.0</Version>
+    <Nullable>enable</Nullable>
+    <TargetFrameworks>net461;netstandard2.0;netstandard2.1</TargetFrameworks>
+    <NoWarn>$(NoWarn);AZC0001;AZC0012</NoWarn>
+    <AssemblyOriginatorKeyFile>Open.snk</AssemblyOriginatorKeyFile>
```

Comment:
If this project is temporary let's use our key.

Reply:
It is not temporary, it needs to match what will ship in dotnet/runtime next release.

Reply:
Gotcha.